src/main/java/com/aventstack/extentreports/Report.java

```diff
@@ -100,8 +100,6 @@ synchronized void addNode(Test node) {
     }
     
     synchronized void addLog(Test test, Log log) {
-        collectRunInfo();
```

Comment:
private void copyNodeAttributeInfoToAttributeContext(Test node) {
        if (node.hasCategory())
            node.getCategoryContext().getAll().forEach(x -> categoryContext.setAttributeContext((Category) x, node));
        
        if (node.hasAuthor())
            node.getAuthorContext().getAll().forEach(x -> authorContext.setAttributeContext((Author) x, node));
        
        if (node.hasChildren())
            node.getNodeContext().getAll().forEach(this::copyNodeAttributeInfoToAttributeContext);
        
        if (node.hasChildren())
            node.getNodeContext().getAll().forEach(x -> {
                copyNodeAttributeInfoToAttributeContext(x);
                //copyNodeExceptionInfoToExceptionContext(x);
            });
    }